apps/cli/cli_auto.c

```diff
@@ -120,6 +120,114 @@ cvec_append(cvec *cvv0,
     return cvv2;
 }
 
+/*! Print an XML tree structure to an output stream and encode chars "<>&"
+ *
+ * @param[in]   xn          clicon xml tree
+ * @param[in]   level       how many spaces to insert before each line
+ * @param[in]   prettyprint insert \n and spaces tomake the xml more readable.
+ * @param[in]   fn          Callback to make print function
+ * @see clicon_xml2cbuf
+ * One can use clicon_xml2cbuf to get common code, but using fprintf is
+ * much faster than using cbuf and then printing that,...
+ */
+int xml2file (cxobj *xn, int level, int prettyprint, clicon_output_cb *fn)
```

Comment:
Please make it static and add something like "autocli_" prefix, eg: static int autocli_xml2file

Reply:
This function is also used in show_cli.c so it can't be static. Do you want to make it an extern function?

Reply:
Oh, did not see that. Fine just please add prefix. No extern please.